cmd/lncli/commands.go

```diff
@@ -1984,7 +1996,7 @@ var updateChannelPolicyCommand = cli.Command{
 
 func parseChanPoint(s string) (*lnrpc.ChannelPoint, error) {
 	split := strings.Split(s, ":")
-	if len(split) != 2 {
+	if len(split) != 2 || len(split[0]) == 0 || len(split[1]) == 0 {
```

Comment:
Do we need this check? Parsing `split[1]` on line 2004 and creating a hash from `split[0]` on line 2009 shoud fail out with more specific errors if they're empty?